compiler+runtime/src/cpp/jank/runtime/obj/persistent_string_sequence.cpp

```diff
@@ -0,0 +1,113 @@
+#include <jank/runtime/obj/persistent_string_sequence.hpp>
+
+namespace jank::runtime
+{
+  obj::persistent_string_sequence::static_object(obj::persistent_string_ptr s)
+    : str{ s }
+  {
+    assert(s->count() > 0);
```

Comment:
clang-tidy will complain about this, since using `!s->empty()` is more idiomatic.